julia-package/brainflow/test/julia_tests.jl

```diff
@@ -0,0 +1,48 @@
+using Test
+using brainflow
+using JSON
+
+@testset "prepare_session error" begin
+    # use a board that is not connected
+    params = brainflow.BrainFlowInputParams()
+    board_shim = brainflow.BoardShim(brainflow.CYTON_BOARD, params)
```

Comment:
I think it should be replaced by Synthetic Board, Cyton will not work wo real hardware. Also prepare without release will lead to memory leak

Reply:
Sorry I misunderstood it, its a test which should fail right? Then its ok

Reply:
Correct